.github/workflows/ios_detox.yml

```diff
@@ -0,0 +1,221 @@
+name: iOS Detox
+
+on: [pull_request]
+
+jobs:
+  detox-build:
+    runs-on: macos-latest
+    timeout-minutes: 60
+
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_11.5.app
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+
+    - name: Generate Detox app cache key
+      run: echo $(git rev-parse HEAD:app) > "./app-git-revision.txt"
+
+    - name: Cache Detox app
+      uses: actions/cache@v1
+      id: detoxappcache
+      with:
+        path: ios/build/Build/Products/Release-iphonesimulator
+        key: iOSDetoxRelease-v4-${{ hashFiles('yarn.lock') }}-${{ hashFiles('ios/Podfile.lock') }}-${{ hashFiles('./app-git-revision.txt') }}
+
+    - name: Node
+      if: steps.detoxappcache.outputs.cache-hit != 'true'
```

Comment:
Is `detoxappcache` correct here? It looks like it should be `node` or `mac-node`